src/Symfony/Component/DomCrawler/Tests/FormTest.php

```diff
@@ -527,6 +533,13 @@ public function testGetUriWithoutAction()
         $this->assertEquals('http://localhost/foo/bar', $form->getUri(), '->getUri() returns path if no action defined');
     }
 
+    public function testGetUriWithActionOverride()
+    {
+        $form = $this->createForm('<form action="/foo"><button type="submit" formaction="/bar" /></form>', null, 'http://localhost/foo/');
+        $this->assertEquals('http://localhost/bar', $form->getUri(), '->getUri() returns absolute URIs');
```

Comment:
Maybe because `getRawUri` is a protected method (can't be tested directly), which is called by public `getUri` method.